source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,48 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-02-24
+
+### Highlights
+
+#### Custom Statuses
+ - Users can set a custom status by using the custom status dropdown in the status menu.
```

Comment:
Custom Statuses allows users to add a descriptive status message and emoji that’s visible to everyone. Users now gain the flexibility to express their current status in any way they prefer. Mobile support is coming soon.